src/_components/accordion/accordion.md

```diff
@@ -74,9 +74,9 @@ Light- eller darkmode tilknyttes via en data-attribute: **data-theme="lightmode"
         <i class="fas fa-chevron-down"></i>
       </div>
       <div class="accordion-body padding-l fontsize-small">
-      <p><b>Lorem Ipsum?</b></p>
+      <div><b>Lorem Ipsum?</b></div>
       Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
-      <p><b>Lorem Ipsum?</b></p>
+      <div class="padding-m--t"><b>Lorem Ipsum?</b></div>
```

Comment:
det korrekte er altså at bruge `<p>` til tekst linjer så må du hellere lave 

```
.accordion-body p {
  STYLING
}
```

Reply:
og det gælder selvfølgelig for alle de steder du har brugt div på en tekst linje i "body"

Reply:
Så skulle den gerne være der